
syntax = "proto3";

service GeneticAnalysis {
    rpc AnalyzeGeneticData (EncryptedGeneticData) returns (EncryptedResponse);
}

message EncryptedGeneticData {
    string encrypted_payload = 1;
}

message EncryptedResponse {
    string encrypted_payload = 1;
}
